src/App.tsx

```diff
@@ -2,73 +1,24 @@
+import React, { useState } from 'react';
 import './App.scss';
 import { peopleFromServer } from './data/people';
+import Autocomplete from './components/Autocomplete';
+import { Person } from './types/Person';
```

Comment:
Ensure that the `Person` type is correctly defined in './types/Person'. Verify that this type matches the expected structure used in the `Autocomplete` component.